pKa_challenge_instructions.md

```diff
@@ -0,0 +1,115 @@
+
+## Experimental details
+pKa measurements were collected using spectrophotometric pKa measurements with Sirius T3 instrument by Mehtap Isik from the Chodera Lab at MSKCC with the support of Merck Rahway, Preformulation Department. 
+
+Small molecules were purchased in powder form. 10 mg/ml DMSO solutions were prepared and used as stock solutions for preparation of samples, where 1-5 uL of 10 mg/ml DMSO stock solution is diluted in 1.5 mL ionic-strength adjusted water (0.15 M KCl). pH titrations with acid (0.5 M HCl, 0.15 M KCl) and base (0.5 M KOH, 0.15 M KCl) and cosolvent addition (80% Methanol, 0.15 M KCl) were performed in an automated fashion with Sirius T3.
+
+[UV-metric pKa measurement protocol of Sirius T3](http://www.sirius-analytical.com/science/pka) evaluates pKas based on change of absorbance in the UV region of the spectrum while a pH titration is being performed between pH 1.8 and 12.2. UV-absorbance spectra of small molecule chromophores can change as a function of pH,  based on the protonation state change of groups close to the chromophore. Change in multiple wavelengths as a function of pH is fitted to a model to determine pKas. Number of pKas in the model is selected based on better fit between experimental and modeled microstates curves. 
+
+This method is capable of measuring pKas between 2 and 12, when protonatable groups are at a maximum distance of 4-5 atoms away from chromophores, so that a change of protonation state affects the absorbance of UV-chromophore. We have selected compounds where titratable groups are close to potential chromophores (conjugated rings), but it is possible to completely miss detection of a pKa value if a titratable group is not in interaction with a UV-chromophore.  
+
+pKa measurements of soluble compounds were performed in ionic-strength adjusted water with 0.15 M KCl. Indications for precipitation was observed by visual inspection of samples and also by inspection of UV-spectra plots as a function of pH at 500-600 nm region of the spectra. For compounds with insufficient solubility, cosolvent protocol is used where 3 UV-metric pKa measurements were done at different cosolvent:water ratios (typically 30%, 40% and 50% methanol) and  Yasuda-Shedlovsky extrapolation method is used to estimate pKa value at 0% cosolvent.
+
+Three replicate pKa measurements were made for all compounds at room temperature (25°C). Multiwavelength absorbance analysis of Sirius T3 allows very good resolution pKas but essentially this method measures macroscopic pKas. Microscopic pKas with with very close pKa values and overlapping changes in absorbance spectra could be measured as one macroscopic pKa value.
```

Comment:
```
Three independent replicate pKa measurements were made for all compounds at room temperature (25°C). 
```
Can you specify how they were replicates? Did you make up independent DMSO stocks, independent dilutions, or just ran three separate aliquots of the same dilution?
```
While the Sirius T3 multiwavelength absorbance analysis allows for precise determination of pKas, this method
 essentially measures macroscopic pKas, rather than microscopic pKas. If the molecule possesses two or more 
microscopic pKas with with very close pKa values and overlapping changes in absorbance spectra, the resulting 
UV shift upon titration could be reported as one macroscopic pKa value.
```

Reply:
Three aliquotes from the same DMSO stock solution were taken to prepare titration samples. There was no intermediate dilution. I added this information to the text.